update.sh

```diff
@@ -0,0 +1,4 @@
+git merge upstream/master
+git pull
+git commit
+git push
```

Comment:
You dont have to worry about this file but for others it pulls from your repository, then theirs and commits it all back to theirs. Mostly it just keeps their fork up to date with yours.